pkg/dashboards/dashboard_test.go

```diff
@@ -174,6 +202,25 @@ func Test_parseUpdateDashboardRequest(t *testing.T) {
 	assert.Equal(t, "mydashboard", details["note"])
 }
 
+// TestCreateDashboardWithDefaultName tests that creating a dashboard with a default name is not allowed
+func TestCreateDashboardWithDefaultName(t *testing.T) {
```

Comment:
Just to be consistent with our other test function naming, this should be:
```suggestion
func Test_CreateDashboardWithDefaultName(t *testing.T) {
```